storybookjs/react-native
# feat: support for storybook 6.4/6.5, CSF3 and HMR fix

Issue:

This is the initial attempts at getting the new story store working in react native storybook

## What I did

- 6.5+ support in rn storybook
- re-uses preview web from storybook core by monkey patching the render functions and url store
- introduces a small document polyfill to stop document.location from crashing in the preview web constructor

## How to test

```
yarn install
cd app/native
yarn prepare
cd ../../examples/native`
yarn web 
```


Early build released at 6.0.1-canary.1

Example project with a early build
https://github.com/dannyhw/react-native-storybook-6.5-example

With the introduction of this patch in 6.5 we should be able to remove the document polyfill
https://github.com/storybookjs/storybook/pull/19623

also resolves #30
